gubernator/view_pr.py

```diff
@@ -68,12 +71,17 @@ def base(path):
 
 class PRHandler(view_base.BaseHandler):
     """Show a list of test runs for a PR."""
-    def get(self, pr):
-        builds = pr_builds(pr)
+    def get(self, path, pr):
+        if path:
+            repo = 'kubernetes/%s' % path.strip('/')
```

Comment:
I guess I'm just confused about whether this handles other orgs at all. If not, that's fine and we can add that support later.